src/app/components/ContactPlaceholder.js

```diff
@@ -134,34 +137,32 @@ const ContactPlaceholder = ({
 
     if (contactGroupID) {
         const { Name } = contactGroups.find(({ ID }) => ID === contactGroupID);
-        const handleClick = () => createModal(<ContactGroupModal contactGroupID={contactGroupID} />);
+        const total = contacts.filter(({ LabelIDs = [] }) => LabelIDs.includes(contactGroupID)).length;
+        const totalContactsText = (
+            <span className="color-primary bold">
+                {total === 1 ? c('Info').t`1 contact` : c('Info').t`${total} contacts`}
+            </span>
+        );
+        console.log(totalContactsText);
+
+        const handleEdit = () => createModal(<ContactGroupModal contactGroupID={contactGroupID} />);
+        const handleExport = () =>
+            createModal(
+                <ExportModal contactGroupID={contactGroupID} userKeysList={userKeysList} disabled={loadingUserKeys} />
```

Comment:
Instead of disabling the `ExportModal` when `loadingUserKeys` is `true`, what do you think of disabling the button calling `handleExport`?

Reply:
I have no idea why that structure was there, since the `ExportModal` is not supposed to take a `disabled` prop. Fixed!